x-pack/plugins/ingest_manager/common/constants/agent.ts

```diff
@@ -14,3 +14,5 @@ export const AGENT_TYPE_TEMPORARY = 'TEMPORARY';
 
 export const AGENT_POLLING_THRESHOLD_MS = 30000;
 export const AGENT_POLLING_INTERVAL = 1000;
+export const AGENT_UPDATE_LAST_CHECKIN_INTERVAL_MS = 30000;
```

Comment:
Should we make these numbers configurable?

Reply:
Making them configurable allow us to tweaks stuff without releasing, until we have all the testing scenarios in place this seems like a good idea. cc @roncohen